task lists/index.html

```diff
@@ -33,11 +35,13 @@
                 </div>
             </div>
             <div class="task__number">
-                <form>
-                    <label>
-                      <input type="checkbox" name="" value="">
-                    </label>
-                </form>
+                <div class="custom__checkbox">
+                    <form>
```

Comment:
Applying the <form> tag for a single checkbox is wrong. You can wrap the whole task list in the form or just remove it from here.